src/lib/pci_virtio_sock.c

```diff
@@ -434,6 +441,25 @@ static size_t iovec_clip(struct iovec **iov, int *iov_len, size_t bytes)
 	return ret;
 }
 
+static size_t iovec_shift_left1(struct iovec **iov, int *iov_len)
```

Comment:
A comment explaining what this is supposed do would be appreciated.

Is it actually just the same as `if (iov_len > 0) iovec_pull(&iov, &iov_len, NULL, iov[0].iov_len)`?